Sources/Commands/Options.swift

```diff
@@ -381,6 +365,26 @@ public struct SwiftToolOptions: ParsableArguments {
         help: "Disable/enable dead code stripping by the linker")
     var linkerDeadStrip: Bool = true
 
+    @Flag(help: "Define automatic transformation of source control based dependencies to registry based ones")
+    var sourceControlToRegistryDependencyTransformation: SourceControlToRegistryDependencyTransformation = .disabled
+
+    enum SourceControlToRegistryDependencyTransformation: EnumerableFlag {
+        case disabled
+        case identity
+        case swizzle
+
+        static func name(for value: Self) -> NameSpecification {
+            switch value {
+            case .disabled:
+                return .customLong("disable-scm-to-regsitry-lookup")
+            case .identity:
+                return .customLong("use-registry-identity-for-scm")
+            case .swizzle:
+                return .customLong("raplace-scm-with-registry")
```

Comment:
```suggestion
                return .customLong("replace-scm-with-registry")
```

Reply:
Is there a way to add help strings for each of the modes?  The longer names are useful but the distinction between the last two seems a bit nuanced.

Reply:
good idea. will do